src/java/fr/paris/lutece/plugins/identitymediation/web/IdentityDuplicateJspBean.java

```diff
@@ -639,13 +639,16 @@ public String doCreateIdentityMergeTask(final HttpServletRequest request) throws
         if(!_canNotify) {
```

Comment:
Problems could occur in a multi-tab context... but lets start with this, and improve later